crates/blockifier/src/execution/syscalls/secp.rs

```diff
@@ -352,3 +365,27 @@ pub fn secp256r1_new(
 ) -> SyscallResult<Secp256r1NewResponse> {
     syscall_handler.secp256r1_hint_processor.secp_new(request)
 }
+
+pub fn allocate_point<Curve: SWCurveConfig>(
+    point_x: U256,
+    point_y: U256,
+    hint_processor: &mut SecpHintProcessor<Curve>,
+) -> cairo_native::starknet::SyscallResult<usize>
+where
```

Comment:
What do you think of putting this on sierra_utils.rs for the time being. I am thinking that as part of the finishing up part we could create a `native_utils` where all native specific utilites are located.

Reply:
good, sounds reasonable